### PR TITLE
Add org.kde.lskat

### PR DESCRIPTION
**Pull request by cho2**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of [permissions](https://invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.lskat.json#L10). 
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
  - I am a member of [KDE Team on Flathub](https://github.com/orgs/flathub/teams/kde/members).
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - KDE project owns the domain
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 
  - n/a

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by cho2** [comment]
bot, build org.kde.lskat

**Comment by cho2** [comment]
cc @flathub/kde

File: org.kde.lskat.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "id": "org.kde.lskat",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "lskat",
+    "rename-icon": "lskat",
+    "finish-args": [
+        "--share=ipc",
```
Comment (travier) [inline org.kde.lskat.json:9]: Likely missing `--device=dri`

**Comment by cho2** [comment]
bot, build org.kde.lskat

**Comment by travier** [comment]
Manifest LGTM. Not yet tested.

**Comment by tsdgeos** [comment]
looks good

**Pull request closed without merging**